samples/sensor/proximity_polling/sample.yaml

```diff
@@ -8,5 +8,4 @@ tests:
       - proximity
     filter: dt_alias_exists("prox-sensor0")
     integration_platforms:
-      - "reel_board_v2" # PHYTEC reel board v2
-      - "reel_board"    # PHYTEC reel board
+      - nrf52840dk_nrf52840
```

Comment:
The reel boards come with a proximity sensor. The nRF52840 DK doesn't?

Reply:
Odd. ` west build --pristine auto -b reel_board_v2 samples/sensor/proximity_polling` builds fine for me.

Reply:
yes, it builds, but in twister, the filter is not matching reel_board at all `filter: dt_alias_exists("prox-sensor0")`

Reply:
```
 ./scripts/twister -T samples/sensor/proximity_polling -Gv
ZEPHYR_BASE unset, using "/home/nashif/zephyrproject/zephyr"
Renaming output directory to /home/nashif/zephyrproject/zephyr/twister-out.2
INFO    - Using Ninja..
INFO    - Zephyr version: v3.4.0-rc2-173-ga8ed28ab6fc8
INFO    - Using 'zephyr' toolchain.
INFO    - Selecting default platforms per test case
INFO    - Building initial testsuite list...
INFO    - Writing JSON report /home/nashif/zephyrproject/zephyr/twister-out/testplan.json
INFO    - JOBS: 112
INFO    - Adding tasks to the queue...
INFO    - Added initial list of jobs to queue
INFO    - 1/2 reel_board_v2             samples/sensor/proximity_polling/sample.sensor.proximity_polling SKIPPED (runtime filter)
INFO    - 2/2 reel_board                samples/sensor/proximity_polling/sample.sensor.proximity_polling SKIPPED (runtime filter)

```

Reply:
this so-called sample is a waste of time, just look at the source code, does this even qualify as a sample? :)

Reply:
True, it does seem a bit thin.